setup.js

```diff
@@ -0,0 +1,6 @@
+const { config } = require('dotenv');
+config();
+
+const client = require('./src/utils/db.js');
+
+client.sync({ force: true })
```

Comment:
Using `sync({ force: true })` will drop and recreate all tables in the database, which can lead to data loss. Make sure this is the intended behavior, especially if this script is run in a production environment.